step.yml

```diff
@@ -2,31 +2,52 @@ title: "Deploy to Bitrise.io - Apps, Logs, Artifacts"
+  4. The **Enable public page for the App?** required input is set to `true` by default. It creates a long and random URL which can be shared with those who do not have a Bitrise account. 
+  If you set this input to `false`, the **Notify: Emails** input will be ignored and the **Notify: User Roles** will receive the build URL instead of the public install URL.
+  5. With the **Compress the artifacts into one file?** required input set to `true`, you can compress the artifacts found in the Deploy directory into a single file.
+  You can specify a custom name for the zip file with the `zip_name` option. Otherwise the default Deploy directory will be used. 
+  If the **Compress the artifacts into one file?** is set to `false`, the artifacts in the Deploy directory will be deployed separately.
+  6. With the **Format for the BITRISE_PUBLIC_INSTALL_PAGE_URL_MAP output** required input field, you can customize the output format of the public install page’s multiple artifact URLs so that the next Step can render the output (for example, our **Send a Slack message** Step). 
+  Provide a language template description using [https://golang.org/pkg/text/template]([https://golang.org/pkg/text/template) so that the **Deploy to Bitrise.io** Step can build the required custom output.
+  7. With the **Format for the BITRISE_PERMANENT_DOWNLOAD_URL_MAP output** required input, you can customize the output format of the `BITRISE_PERMANENT_DOWNLOAD_URL_MAP` so that the next Step can render the output. 
+  This input creates a permanent URL containing all the artifacts the build has generated.
+  Provide a language template description using [https://golang.org/pkg/text/template]([https://golang.org/pkg/text/template) so that the **Deploy to Bitrise.io** Step can build the required custom output.
+  8. The **Test API's base URL** and the **API Token** input fields are automatically populated for you.
+  
+  #### Configuring the Debug section of the Step
+  
+  If you wish to use any of the Step’s debug features, set the following inputs:
+  1. In the **Name of the compressed artifact (without .zip extension)** input you can add a custom name for the compressed artifact. If you leave this input empty, the default `Deploy directory` name is used.
+  Please note that this input only works if you set the **Compress the artifacts into one file?** input to `true`.
+  2. The **Bitrise Build URL** and the **Bitrise Build API Token** inputs are automatically populated.
+  3. If **The Enable Debug Mode** required input is set to `true`, the Step prints more verbose logs. It is `false` by default. 
+  4. If you wish to generate an Android App Bundle and an APK which you can test, set a specific [bundletool version]((https://github.com/google/bundletool/releases) in the **Bundletool version** required input.  
```

Comment:
The bundletool is used for generating the APK from the AAB. For the aab generation it is not needed. This input is required so even if you don't set it we will use the default bundletool when we see an AAB.

I think currently the description implies that if you need AAB you have to change the value here somehow.